### PR TITLE
docs: hide private methods from docs

### PR DESCRIPTION
**Pull request by w3b6x9**
## What kind of change does this PR introduce?

docs update

## What is the new behavior?

Private methods do not show up in docs.

### CODE REVIEW

File: src/RealtimeChannel.ts [hunk 1]
```diff
@@ -76,6 +81,7 @@ export default class RealtimeChannel {
   pushBuffer: Push[] = []
   presence: RealtimePresence
 
+  /** Topic name can be any string. */
```
Comment (alaister) [inline src/RealtimeChannel.ts:84]: Should this be one line down? Right above `public topic: string`?

**Review by alaister — CHANGES_REQUESTED** [review]
Nice! Just one tiny change I think

File: src/RealtimeChannel.ts (hunk 1 above)
Comment (w3b6x9) [reply src/RealtimeChannel.ts:84]: @alaister I think you're right. Will address.

Comment (w3b6x9) [reply src/RealtimeChannel.ts]: https://github.com/supabase/realtime-js/pull/202/commits/0c567f9a9de1a4cc257229705ebfb4a8c85b2efe

**Review by alaister — APPROVED** [review]
Thanks @w3b6x9!

**Pull request merged**